tests/init.go

```diff
@@ -35,12 +35,43 @@ var Forks = map[string]*params.ChainConfig{
 	"Frontier": {
 		ChainID: big.NewInt(1),
 	},
+	"Homestead": {
+		ChainID: big.NewInt(1),
+	},
 	"Byzantium": {
 		ChainID: big.NewInt(1),
 	},
 	"Constantinople": {
 		ChainID: big.NewInt(1),
 	},
+	"ConstantinopleFix": {
+		ChainID: big.NewInt(1),
+	},
+	"Istanbul": {
+		ChainID: big.NewInt(1),
+	},
+	"Berlin": {
+		ChainID: big.NewInt(1),
+	},
+	"London": {
+		ChainID: big.NewInt(1),
+	},
+	"Paris": {
+		ChainID:                  big.NewInt(1),
+		IstanbulCompatibleBlock:  new(big.Int),
+		LondonCompatibleBlock:    new(big.Int),
+		EthTxTypeCompatibleBlock: new(big.Int),
+		MagmaCompatibleBlock:     new(big.Int),
+		KoreCompatibleBlock:      new(big.Int),
+	},
+	"Merge": {
```

Comment:
It seems not related to Kaia. Can this be omited?

Reply:
Merge is used. On the other hand, Paris wasn't used so I'll remove Paris (AFAIK, Paris is equivalent to Merge.).

Reply:
Did you mean `Merge` as Ethereum's 'The Merge'? If so, I wonder if those tests are effective in Kaia.

Reply:
Yes.

Actually, this is just the config in the case where Kaia behaves as Ethereum Merge. This means that we can say Kaia has Ethereum Merge compatibility if tests pass using this config.